src/locales/de_DE/de_DE.json

```diff
@@ -343,7 +343,7 @@
         "title_selfie": "Selfie"
     },
     "poa_guidance": {
-        "button_primary": "Fortfahren",
+        "button_primary": "Continue",
```

Comment:
Is this change intentional?

Reply:
Yes, PoA does not support english at the moment, so for consistency the strings that were translated by mistake were reverted to English